src/main/java/com/baidu/hugegraph/structure/schema/VertexLabel.java

```diff
@@ -78,7 +78,7 @@ public String toString() {
 
         Builder useCustomizeNumberId();
 
-        Builder useCustomizeUuid();
```

Comment:
also update IdStrategy.isCustomize() method to 
```
this == IdStrategy.CUSTOMIZE_STRING || this == IdStrategy.CUSTOMIZE_NUMBER || this == IdStrategy.CUSTOMIZE_UUID;

```

and please add method in IdStrategy.class

```
public boolean isCustomizeUUID() {
        return this == IdStrategy.CUSTOMIZE_UUID;
    }
```